e2e/process-services-cloud/start-task-form-cloud.e2e.ts

```diff
@@ -208,4 +260,289 @@ describe('Start Task Form', () => {
         });
 
     });
+
+    describe('Attach content to process-cloud task form using upload widget', async () => {
+
+        beforeEach(async (done) => {
+            navigationBarPage.navigateToProcessServicesCloudPage();
+            appListCloudComponent.checkApsContainer();
+            appListCloudComponent.checkAppIsDisplayed(candidateBaseApp);
+            appListCloudComponent.goToApp(candidateBaseApp);
+            processCloudDemoPage.openNewProcessForm();
+            done();
+        });
+
+        it('[C310358] Should be able to attach a file to a form from local', async () => {
+            startProcessPage.clearField(startProcessPage.processNameInput);
```

Comment:
Instead of starting the process from UI, isn't it possible to start it using API calls?